### PR TITLE
Update packages via npm and modify page2 with attributes

### PR DESCRIPTION
**Pull request by alphaX86**
Hello Devfolio! 👋 I've updated all packages by running the command `npm update` to latest versions of GatsbyJS, React, React-DOM and more!

Also, I made changes in the page 2 by making it look good (added Header and a button for go back)

Hope you like it 😉

### CODE REVIEW

File: src/pages/page-2.js [hunk 1]
```diff
@@ -2,14 +2,20 @@ import React from 'react';
...
+    <Navbar />
     <Container>
       <h1>Hi from the second page</h1>
       <p>Welcome to page 2</p>
-      <Link to="/">Go back to the homepage</Link>
+      <p>Feel free to go back to Homepage! :)</p>
+      <Button as={Link} to="/" style={{ textDecoration: 'none' }}>
+            Go to Homepage &nbsp; &larr;
+          </Button>
     </Container>
```
Comment (alphaX86) [inline src/pages/page-2.js:19]: Changes made in Page 2

**Comment by alphaX86** [comment]
Unnecessary packages are removed and existing packages are updated to latest versions

**Comment by alphaX86** [comment]
Oh, and it works! So don't worry! You can even test and check! 😄

**Comment by apollonian** [comment]
Thanks, @alphaX86! Will take a look.

**Comment by alphaX86** [comment]
Seems you have conflicts on the repo... I couldn't check them! Can you do that?

**Pull request closed without merging**